CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # Changelog
 
+## 0.6.0
+
+There have been 31 releases and 1135 pull requests since [0.5.0](https://github.com/astral-sh/uv/releases/tag/0.5.0), our last release with breaking changes. As before, we've accumulated various changes that improve correctness and user experience, but could break some workflows. This release contains those changes; many have been marked as breaking out of an abundance of caution. We expect most users to be able to upgrade without making changes.
+
+### Breaking changes
+
+- **Create `main.py` instead of `hello.py` in `uv init`** ([#10369](https://github.com/astral-sh/uv/pull/10369))
+
+  Previously, `uv init` created a `hello.py` sample file. Now, `uv init` will create `main.py` instead — which aligns with expectations from user feedback. The `--bare` option can be used to avoid creating the file. 
+
+- **Respect `UV_PYTHON` in `uv python install`** ([#11487](https://github.com/astral-sh/uv/pull/11487))
+
+  Previously, `uv python install` did not read this environment variable; now it does. We believe this matches user expectations, however, this will take priority over `.python-version` files which could be considered breaking.
+
+- **Set `UV` to the uv executable path** ([#11326](https://github.com/astral-sh/uv/pull/11326))
```

Comment:
(I actually didn't follow this, what's it for?)

Reply:
Ah this was for https://github.com/astral-sh/uv/issues/8775 and the Ray team